telemetry/jetbrains/src/main/kotlin/software/aws/toolkits/telemetry/generator/gradle/GenerateTelemetry.kt

```diff
@@ -21,7 +21,7 @@ open class GenerateTelemetry : DefaultTask() {
     fun generate() {
         println("Generating telemetry using packaged file and:\n ${inputFiles.map { it.absolutePath }.joinToString("\n")}")
         try {
-            TelemetryGenerator.generateTelemetryFromFiles(inputFiles, outputDirectory)
+            TelemetryGenerator.generateTelemetryFromFiles(inputFiles, outputFolder = outputDirectory)
```

Comment:
Is this change necessary if you don't reorder the parameters in `generateTelemetryFromFiles`?

Reply:
No, but I think the parameter reorder makes more sense in my mind. I'll change these calls to all use named parameters.